tgui/packages/tgui/interfaces/StripMenu.tsx

```diff
@@ -6,7 +6,14 @@ import { Box, Button, Icon, Stack } from '../components';
 import { Window } from '../layouts';
 
 const ROWS = 5;
-const COLUMNS = 5;
+const COLUMNS = 9;
+
+const getColumnsAmount = (mode: number): number => {
+  if (mode === 0) {
+    return 5;
+  }
+  return 9;
```

Comment:
wouldn't it be better to do `return ROWS` and `return COLUMNS` instead of just hardcoding the magic numbers?

Reply:
no because its not `ROWS` and `COLUMNS`
these are two options for `COLUMNS` in two modes

Reply:
i see that my ts code is not good but i dont quite know that lang and i dont know how to make it better to support two modes in another way